tableau-server-single-node/scripts/config-linux.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# confirmed: this script works when run manually as root user from root top directory using the following command
+# sh ./config-linux.sh -u <username> -p <password> -h admin -i admin -j 98107 -k usa -l seattle -m data -n tech -o yes -q pm -r 8888888 -s tableau -t wa -v dev -w jamie -x jdata@tableau.com [-y <license key>]
+# customized to reflect machine admin username and admin password
+
+while getopts u:p:h:i:j:k:l:m:n:o:q:r:s:t:v:w:x:y: option
+do
+ case "${option}"
+ in
+ u) USER=${OPTARG};;
+ p) PASSWORD=${OPTARG};;
+ h) TS_USER=${OPTARG};;
+ i) TS_PASS=${OPTARG};;
+ j) ZIP=${OPTARG};;
+ k) COUNTRY=${OPTARG};;
+ l) CITY=${OPTARG};;
+ m) LAST_NAME=${OPTARG};;
+ n) INDUSTRY=${OPTARG};;
+ o) EULA=${OPTARG};;
+ q) TITLE=${OPTARG};;
+ r) PHONE=${OPTARG};;
+ s) COMPANY=${OPTARG};;
+ t) STATE=${OPTARG};;
+ v) DEPARMENT=${OPTARG};;
+ w) FIRST_NAME=${OPTARG};;
+ x) EMAIL=${OPTARG};;
+ y) LICENSE_KEY=${OPTARG};;
+esac
+done
+
+cd /tmp/
+
+# create secrets
+echo "tsm_admin_user=\"$USER\"\ntsm_admin_pass=\"$PASSWORD\"\ntableau_server_admin_user=\"$TS_USER\"\ntableau_server_admin_pass=\"$TS_PASS\"" >> secrets
+
+# create registration file
+echo "{
+ \"zip\" : \"$ZIP\",
+ \"country\" : \"$COUNTRY\",
+ \"city\" : \"$CITY\",
+ \"last_name\" : \"$LAST_NAME\",
+ \"industry\" : \"$INDUSTRY\",
+ \"eula\" : \"$EULA\",
+ \"title\" : \"$TITLE\",
+ \"phone\" : \"$PHONE\",
+ \"company\" : \"$COMPANY\",
+ \"state\" : \"$STATE\",
+ \"department\" : \"$DEPARMENT\",
+ \"first_name\" : \"$FIRST_NAME\",
+ \"email\" : \"$EMAIL\"
+}" >> registration.json
+
+# create config file
+echo '{
+  "configEntities": {
+    "identityStore": {
+      "_type": "identityStoreType",
+      "type": "local"
+    }
+  }
+}' >> config.json
+wait
+
+# download tableau server .deb file
+wget --output-document=tableau-installer.deb https://downloads.tableau.com/esdalt/2019.1.2/tableau-server-2019-1-2_amd64.deb
+
+# download automated-installer
+wget --remote-encoding=UTF-8 --output-document=automated-installer.sh https://raw.githubusercontent.com/tableau/server-install-script-samples/master/linux/automated-installer/automated-installer
```

Comment:
move a copy of this to this repo/sample

Reply:
@kkader

Reply:
@bmoore-msft This script is maintained by another Tableau team.  They periodically update it but the link remains the same.  I'd prefer to link to is directly instead of maintaining a copy in this repo.  Does that work?

Reply:
No we don't want linking across repos to raw source - too fragile.  If they have a release asset you can link to a fixed version, other than that you need to copy here.